lib/src/value/calculation.dart

```diff
@@ -223,6 +643,63 @@ class SassCalculation extends Value {
   /// simplification.
   SassCalculation._(this.name, this.arguments);
 
+  // Returns [value] coerced to [number]'s units.
+  static SassNumber _matchUnits(double value, SassNumber number) {
+    return SassNumber.withUnits(value, numeratorUnits: number.numeratorUnits);
+  }
```

Comment:
This implementation is wrong: it won't match `number`'s denominator units if it has them.

Also, use `=>` for functions whose body is a single expression.